src/halmos/sevm.py

```diff
@@ -755,10 +757,16 @@ def extend_path(self, path):
         self.extend(path.conditions.keys())
 
 
+@dataclass
+class StorageData:
+    symbolic: bool = False
+    mapping: dict = field(default_factory=dict)
+
+
 class Exec:  # an execution path
     # network
     code: dict[Address, Contract]
-    storage: dict[Address, dict[int, Any]]  # address -> { storage slot -> value }
+    storage: dict[Address, StorageData]  # address -> { storage slot -> value }
```

Comment:
would a `defaultdict` work here? we wouldn't have to check if keys exist and explicitly initialize them

Reply:
yes, done in #356